Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/GifViewController.swift

```diff
@@ -0,0 +1,383 @@
+//
+//  EmojiMainViewController.swift
+//  StreamChatUI
+//
+//  Created by Mohammed Hanif on 26/04/22.
+//
+
+import UIKit
+import GiphyUISDK
+import Combine
+
+@available(iOS 13.0, *)
+class GifViewController: UIViewController {
```

Comment:
@mdhanif-simformsolutions  Should use MVVM architecture.

Reply:
Updated this with MVVm Arch. and combine.